include/boost/beast/core/impl/multi_buffer.hpp

```diff
@@ -916,7 +916,7 @@ prepare(size_type n) ->
             std::size_t altn = in_size_ * growth_factor;
 	    // Overflow detection:
             if(in_size_ > altn)
-                altn = std::numeric_limits<std::size_t>::max();
+                altn = (std::numeric_limits<std::size_t>::max)();
```

Comment:
Maybe do a search for `"max("` (and `"min("`) in all files?

Reply:
You must be looking at an old commit?
I have 
```
            if(in_size_ > altn)
                altn = (std::numeric_limits<std::size_t>::max)();
```

Reply:
Ah I see. it was just a comment.